db/queries/exec.sql

```diff
@@ -42,12 +42,12 @@ update unweave.exec
 set metadata = jsonb_set(metadata, '{connection_info}', @connection_info::jsonb)
 where id = $1;
 
--- name: ExecsGet :many
+-- name: ExecsGetActive :many
 select *
 from unweave.exec
-where project_id = $1
+where project_id = $1 and (status <> 'terminated') and provider = $2
 order by unweave.exec.created_at desc
-limit $2 offset $3;
+limit $3 offset $4;
```

Comment:
I don't think we want this. You should be able to get an exec without filtering it by active status

Reply:
Yeah this was a bit of a patch for the CLI. This was because every one of the terminated execs was showing in my list of execs in SSH, and, I think that in order for pagination to work properly that filtering has to be done on the DB rather than in memory. Otherwise you end up in a situation where 1000 terminated execs and one active Exec would yield no results. There’s definitely a better approach for this by having a new query, should have undone this before committing.